antora.yml

```diff
@@ -98,4 +98,5 @@ asciidoc:
     prod2: Eclipse&#160;Che
     project-context: che
     repo-path: ""
+    rh-os-devspaces: Red Hat OpenShift Dev Spaces
```

Comment:
Might want a similar variable for `Red Hat OpenShift Local`

Reply:
Okay, I added it. No idea what happened there and thank you for your reviews!